cypress/page/login.page.ts

```diff
@@ -0,0 +1,26 @@
+class LoginPage {
+    private loginURL: string;
+    private userNameField: string;
+    private passwordField: string;
+    private loginButton: string;
+  
+    constructor() {
+      this.loginURL = "http://saucedemo.com/";
+      this.userNameField = ".login-box > form > div > input#user-name";
+      this.passwordField = ".login-box > form > div > input#password";
+      this.loginButton = "input[type='submit']";
+    }
+  
+    public visitLoginPage(): void {
+      cy.visit(this.loginURL);
+    }
+  
+    public signIn(): void {
+      cy.get(this.userNameField).type("standard_user");
+      cy.get(this.passwordField).type("secret_sauce");
```

Comment:
Please try to avoid to put directly the user and password. Instead, receive them as parameters